packages/core/content-releases/admin/src/components/CMReleasesContainer.tsx

```diff
@@ -138,42 +140,67 @@ const AddActionToReleaseModal = ({
         {({ values, setFieldValue }) => {
           return (
             <Form>
-              <ModalBody>
-                <Flex direction="column" alignItems="stretch" gap={2}>
-                  <Box paddingBottom={6}>
-                    <SingleSelect
-                      required
-                      label={formatMessage({
-                        id: 'content-releases.content-manager-edit-view.add-to-release.select-label',
-                        defaultMessage: 'Select a release',
+              {releases?.length === 0 ? (
+                <NoContent
+                  content={{
+                    id: 'content-releases.content-manager-edit-view.add-to-release.no-releases-message',
+                    defaultMessage:
+                      'No available releases. Open the list of releases and create a new one from there.',
+                  }}
+                  action={
```

Comment:
Since AddActionToReleaseModal is already pretty long, I would recommend to extract NoContent and its children to another component (still in this file). I think it can make the code more readable